### PR TITLE
Falco

### PR DESCRIPTION
**Pull request by FuhuXia**
For https://github.com/GSA/data.gov/issues/3799
1. Install falco helm chart with falcosidekick
2. Add slack webhookurl. `slack_webhookurl` needs to be added to local file `terraform.tfvars` to have slack notification. 

Custom Falco rule added for solr security but havn't figure out a way to apply it in the terraform code. remain in draft for now until coming back to fix it.

### CODE REVIEW

File: terraform/modules/provision-k8s/k8s-falco.tf [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Creating namespace for falco.
+# So that the default-deny-egress network policy does not affect falco pods.
+resource "kubernetes_namespace" "falco" {
+  metadata {
+    name = "falco"
+  }
+}
```
Comment (nickumia-reisys) [inline terraform/modules/provision-k8s/k8s-falco.tf:7]: I would suggest you annotate the namespace to help identify it better in terms of connecting to other resources, k8s relies a lot on annotations to know how to operate in different settings and environments.  If we wanted to create a specific network policy for falco, the annotation would probably come in handy,
Example: https://github.com/GSA/datagov-brokerpak-eks/blob/main/terraform/modules/provision-k8s/k8s-network-policy.tf#L2-L9

**Review by nickumia-reisys — COMMENTED** [review]
I know we said that there isn't a good way of testing this (or at least a nice way of testing it).  We should probably update the README to mention that Falco is installed and doing something.

**Comment by nickumia-reisys** [comment]
Also update the terraform.tfvars instructions on configuring the slack_webhookurl,
- https://github.com/GSA/datagov-brokerpak-eks/blob/main/terraform/modules/provision-aws/terraform.tfvars-template
- https://github.com/GSA/datagov-brokerpak-eks/blob/main/terraform/modules/provision-aws/README.md

**Comment by btylerburton** [comment]
close this?

**Pull request closed without merging**